src/Aspire.Hosting.Keycloak/KeycloakResouceBuilderExtensions.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Keycloak;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Keycloak resources to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class KeycloakResouceBuilderExtensions
```

Comment:
I think there is a typo here Keycloak*Resouce*BuilderExtensions should be Keycloak**Resource**BuilderExtensions.